etc/neutron/services/f5/esd/demo.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
I would say that we should not touch this particular file since it is shipped with the product.  I realize that you are using this for test service definitions.  I would propose that you create a new file in the testdata directory used by other neutronless tests.

Reply:
Acknowledged, will do.

Reply:
Done.